packages/taiko-client/driver/preconf_blocks/api.go

```diff
@@ -307,10 +317,20 @@ func (s *PreconfBlockAPIServer) GetStatus(c echo.Context) error {
 	s.lookaheadMutex.Lock()
 	defer s.lookaheadMutex.Unlock()
 
+	endOfSequencingBlockHash := common.Hash{}
+
+	if s.rpc.L1Beacon == nil {
```

Comment:
```suggestion
	if s.rpc.L1Beacon != nil {
```

A typo here! Only if the L1 beacon client is NOT nil we should run the code below.